doc/source/examples.rst

```diff
@@ -105,8 +105,8 @@ pyndl.ndl.ndl
 :py:mod:`pyndl.ndl.ndl` is a parallel Python implementation using numpy,
 multithreading and a binary format which is created automatically. It allows
 you to choose between the two methods ``openmp`` and ``threading``, with the
-former one using `openMP <http://www.openmp.org/>`_ and therefore being expected
-to be much faster when analyzing larger data. Besides, you can set three
+former one using `openMP <http://www.openmp.org/>`_ and therefore being
+expected to be faster when analyzing larger data. Besides, you can set three
```

Comment:
We should add a sentence here that OpenMP is Linux only and other OS should use threading to make clear why threading here is used instead of openmp

Reply:
Something along the lines:
> Unfortunately, ``method=openmp`` is only available on Linux right now, therefore all examples use ``method='threading'``.
Can you add this into the text? Thanks!